subcommand/server-acl-init/command.go

```diff
@@ -547,37 +607,57 @@ func (c *Command) setServerTokens(logger hclog.Logger, consulClient *api.Client,
 	return nil
 }
 
+func (c *Command) setServerPolicy(consulClient *api.Client, logger hclog.Logger) (api.ACLPolicy, error) {
+	agentRules, err := c.agentRules()
+	if err != nil {
+		logger.Error("Error templating server agent rules", "err", err)
+		return api.ACLPolicy{}, err
```

Comment:
There is a whole lot of inconsistency in the whole system about where logging happens. It would be good to file an issue and take a comprehensive look at in the future.